src/main/java/org/geysermc/cumulus/response/impl/CustomFormResponseImpl.java

```diff
@@ -160,11 +184,10 @@ public <T> T next() {
 
   @Override
   public int asDropdown() {
-    Object next = next();
-    if (next == null) {
+    Object next = nextOrAbsent();
+    if (next instanceof AbsentComponent) {
       return 0;
-    }
-    if (next instanceof Integer) {
+    } else if (next instanceof Integer) {
```

Comment:
Can you remove the `else` from the `else if`?
The else is not needed because the `if` always returns.

(Please apply this comment for the whole class)